app/src/main/java/com/example/sharingang/users/InMemoryUserRepository.kt

```diff
@@ -42,4 +42,8 @@ class InMemoryUserRepository : UserRepository {
         usersMap[user.id] = user
         return true
     }
+
+    override suspend fun report(reportedUser: User, reporterUser: User): Boolean {
+        return false
```

Comment:
I think that even though we're doing nothing, we can return `true` so the UI will work correctly and display the success message

Reply:
Got it!